README.md

```diff
@@ -1,29 +1,29 @@
 # Unified Catalog Migration
 
-Tool for migrating your Unified Catalog items and assets to Marketplace products.
+A tool for migrating your Unified Catalog items and assets to Marketplace products.
```

Comment:
A tool for migrating your Unified Catalog items and assets to the Amplify Enterprise Marketplace.